tests/MiniCover.UnitTests/MiniCover.UnitTests.csproj

```diff
@@ -12,5 +12,6 @@
   </ItemGroup>
   <ItemGroup>
     <ProjectReference Include="..\..\src\MiniCover\MiniCover.csproj" />
+    <ProjectReference Include="..\..\src\MiniCover.HitServices\MiniCover.HitServices.csproj" />
```

Comment:
Because of PrivateAssets above now we need to reference this project, because it's dll is not copied through MiniCover project reference.